app/seeds/admin_seeder.rb

```diff
@@ -0,0 +1,33 @@
+class AdminSeeder
+  ADMINS = [
+    {
+    first_name: "Meder",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.